docs/reference/api/docker_remote_api_v1.20.md

```diff
@@ -140,9 +140,14 @@ Create a container
                    "com.example.license": "GPL",
                    "com.example.version": "1.0"
            },
-           "Volumes": {
-                   "/tmp": {}
-           },
+           "Mounts": [
+             {
+               "Source": "/data",
+               "Destination": "/data",
+               "Relabel": "ro,Z",
```

Comment:
I believe it should be renamed to mode.  The patch I submitted to do :u changes Relabel to Mode.